### PR TITLE
Install Redux

### PR DESCRIPTION
**Pull request by joramot**
**### Project requirements**

- Added Redux Toolkit.
- Created directory Redux.
- Configured a Redux store.
- Defined a slice of state for books, including:
        an array of books (initial state: empty array)
        a reducer that adds a book.
        a reducer that removes a book.

- Defined a slice of state for categories, including:
        an array of categories.
         a reducer that checks the status and always returns "Under construction".

- Structured the application files using a "feature folder" approach.

### CODE REVIEW

File: src/redux/books/booksSlice.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+      ...state,
+      books: [...state.books, action.payload],
+    }),
+    removeBook: (state, action) => ({
+      ...state,
+      books: state.books.filter((book) => book.id !== action.payload),
+    }),
+  },
+});
+
```
Comment (t-yanick) [inline src/redux/books/booksSlice.js:21]: - [ ] Great job working on this milestone. As per project requirements, you should export addBook and removeBook as actions
`export const { addBook, removeBook } = booksSlice.actions`

File: src/redux/categories/categoriesSlide.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  name: 'categories',
+  initialState,
+  reducers: {
+    checkStatus: (state) => ({
+      ...state,
+      categories: ['Under construction"'],
+    }),
+  },
+});
+
```
Comment (t-yanick) [inline src/redux/categories/categoriesSlide.js:17]: - [ ] As with the booksSlice.js, you should also export `checkStatus` as an action.
`export const { checkStatus } = categoriesSlice.actions`

**Review by t-yanick — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED

Hi @joramot , 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  `@reduxjs/toolkit` & `react-redux` present in `package.json`.
:heavy_check_mark:  The structure of the files follow the `ducks pattern`
:heavy_check_mark: Descriptive Readme file
:heavy_check_mark:  Use of correct gitflow
:heavy_check_mark:  Linters all passing
:heavy_check_mark:  Good commit messages
:heavy_check_mark:  All required reducers added

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by danmainah — APPROVED** [review]
Hi @joramot , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights
✔️ Implemented the changes requested to ensure the actions are being exported.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**